code/datums/id_trim/jobs.dm

```diff
@@ -361,8 +361,8 @@
 	trim_state = "trim_paramedic"
 	sechud_icon_state = SECHUD_PARAMEDIC
 	extra_access = list(ACCESS_SURGERY)
-	minimal_access = list(ACCESS_AUX_BASE, ACCESS_CARGO, ACCESS_CONSTRUCTION, ACCESS_ENGINE, ACCESS_EVA, ACCESS_HYDROPONICS,
-					ACCESS_MAINT_TUNNELS, ACCESS_MECH_MEDICAL, ACCESS_MEDICAL, ACCESS_MINERAL_STOREROOM, ACCESS_MORGUE, ACCESS_RESEARCH)
+	minimal_access = list(ACCESS_CONSTRUCTION, ACCESS_HYDROPONICS, ACCESS_MAINT_TUNNELS, ACCESS_MECH_MEDICAL,
```

Comment:
We really need to make these all multi-lined lists by the way these diffs are annoying to read